magefiles/helm.go

```diff
@@ -0,0 +1,110 @@
+//go:build mage_helm
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/magefile/mage/sh"
+	"golang.org/x/xerrors"
+	"gopkg.in/yaml.v3"
+)
+
+const chartFile = "./helm/trivy/Chart.yaml"
+
+func main() {
+	trivyVersion, err := version()
+	if err != nil {
+		log.Fatalf("could not determine Trivy version: %v", err)
+	}
+	input, err := os.ReadFile(chartFile)
+	if err != nil {
+		log.Fatalf("could not find helm chart %s: %v", chartFile, err)
+	}
+	jsonData := map[string]interface{}{}
+	if err := yaml.Unmarshal(input, &jsonData); err != nil {
+		log.Fatalf("could not unmarshal helm chart %s: %v", chartFile, err)
+	}
+	currentAppVersion, ok := jsonData["appVersion"].(string)
+	if !ok {
+		log.Fatalf("could not determine current app version")
+	}
+	currentHelmVersion, ok := jsonData["version"].(string)
+	if !ok {
+		log.Fatalf("could not determine current helm version")
+	}
+	newHelmVersion := newHelmVersion(currentHelmVersion, currentAppVersion, trivyVersion)
+
+	log.Printf("Current helm version %q with Trivy %q will bump up %q with Trivy %q",
+		currentHelmVersion, currentAppVersion, newHelmVersion, trivyVersion)
+
+	newBranch := fmt.Sprintf("ci/helm-chart/bump-trivy-to-%s", trivyVersion)
+	title := fmt.Sprintf("ci(helm): bump Trivy version to %s", trivyVersion)
+	description := fmt.Sprintf("This PR bumps Trivy up to the %s version for the Helm chart.", trivyVersion)
```

Comment:
It might make sense to write about change in `helm` version as well.

Reply:
done https://github.com/aquasecurity/trivy/pull/7526/commits/5a43735fabd3d5b22cff4f3b6923ebb72edb9e8e